src/dev/jest/integration_tests/junit_reporter.test.js

```diff
@@ -78,16 +81,14 @@ it(
                 $: {
                   classname: 'Jest Tests.·',
                   name: 'fails',
-                  time: expect.anything()
+                  time: expect.anything(),
                 },
-                failure: [
-                  expect.stringMatching(/Error: failure\s+at /m)
-                ]
-              }
-            ]
-          }
-        ]
-      }
+                failure: [expect.stringMatching(/Error: failure\s+at /m)],
+              },
```

Comment:
Looks like you have the prettier plugin installed in your editor. If you turn that off and only rely on the eslint plugin it will only apply prettier to files which are supposed to be prettierified.